src/components/dev-hub/community-champion-form.js

```diff
@@ -175,7 +199,11 @@ const Form = React.memo(({ setSuccess, success }) => {
                 variant="default"
             >
                 {experienceOptions.map((option, index) => (
-                    <Radio key={index} value={option}>
+                    <Radio
+                        checked={experience == option}
```

Comment:
adding `checked` attribute resolves this warning I was having when switching between radio buttons: 

<img width="550" alt="Screen Shot 2021-07-28 at 12 15 09 PM" src="https://user-images.githubusercontent.com/46455728/127358804-e7e0d3d1-bc91-4f4b-86c0-65ef1662551f.png">

Reply:
yess! forgot that was a thing haha